nixos/modules/services/continuous-integration/gitlab-runner.nix

```diff
@@ -116,10 +116,14 @@ in
         ++ optional hasDocker "docker.service";
       requires = optional hasDocker "docker.service";
       wantedBy = [ "multi-user.target" ];
+      preStart = "cp ${configFile} /var/lib/gitlab-runner/config.toml";
+      reload = "cp ${configFile} /var/lib/gitlab-runner/config.toml";
```

Comment:
Wouldn't this also work with changing a symlink? Having a copy makes it harder to see where the settings come from.

Reply:
If i remember correctly then this file will be updated with the token of the gitlab ci master